tests/mbt/model/ccv.qnt

```diff
@@ -93,6 +99,30 @@ module ccv_types {
             // a monotonic strictly increasing and positive ID that is used
             // to uniquely identify the Vscs sent to the consumer chains.
             runningVscId: int,
+
+            // For every consumer chain, stores the consumer address assigned by each validator.
+            validatorConsumerAddr: Chain -> (Node -> ConsumerAddr),
```

Comment:
While reading the rest of the code, I had to come back to see again what exactly `validatorConsumerAddr` is. I think it might helpfult to rename to something like `validatorToConsumerAddr` and rename `validatorByConsumerAddr: Chain -> (ConsumerAddr -> Node),` to something like `consumerAddrToValidator`?
Or even (might be a bit too verbose) something like `chainToConsumerAddrToValidator`. I'm not sure if Quint has any naming conventions for this.

Reply:
I think the only convention is the one we come up with :) I went with your first suggestion